admin/config.yml

```diff
@@ -132,7 +132,7 @@ collections:
             - { label: 'Subheading 3', name: 'subheading3', widget: 'string', required: false, i18n: true}
             - { label: 'Column 3', name: 'col3', widget: 'markdown', required: false, i18n: true }
       - label: 'Steps Page'
-        name: 'steps'
+        name: 'component'
```

Comment:
It looks like the name needs to be unique? "Two Column" also uses "component" as a name.

The admin page isn't loading on Federalist:

https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.app.cloud.gov/preview/18f/identity-site/jd-LG-4619-create-an-account-header/admin/#/